ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -124,385 +129,392 @@ Control {
-                        asset.emoji: model.emoji ?? ""
-                        asset.name: asset.emoji || "filled-account"
-                        asset.isLetterIdenticon: asset.emoji
-                        asset.letterSize: 14
-                        asset.bgColor: Theme.palette.primaryColor3
-                        asset.isImage: asset.emoji
-                        enabledNetworks: root.enabledNetworks
-                        rightSideButtons: RowLayout {
-                            StatusFlatRoundButton {
-                                Layout.preferredWidth: 32
-                                Layout.preferredHeight: 32
-                                type: StatusFlatRoundButton.Type.Secondary
-                                icon.name: "send"
-                                icon.color: Theme.palette.baseColor1
-                                enabled: root.sendToAccountEnabled
-                                onClicked: {
-                                    Global.openSendModal(model.address)
-                                }
+            StatusGridView {
+                id: accountsView
+                width: 606
+                height: parent.height
+                anchors.top: parent.top
+                anchors.topMargin: (webView.contentY > Style.current.halfPadding) ? 1 : Style.current.bigPadding
+                Behavior on anchors.topMargin { NumberAnimation { duration: 50 } }
+                anchors.bottom: parent.bottom
+                anchors.bottomMargin: Style.current.halfPadding
+                anchors.horizontalCenter: parent.horizontalCenter
+                anchors.horizontalCenterOffset: Style.current.halfPadding
+                cellWidth: d.delegateWidthM
+                cellHeight: d.delegateHeightM
+                visible: count
+                clip: false
+                ScrollBar.vertical: StatusScrollBar { anchors.right: parent.right; anchors.rightMargin: -3.5 }
+                model: accountsProxyModel
+                delegate: InfoCard {
+                    id: accountInfoDelegate
+                    implicitWidth: GridView.view.cellWidth - Style.current.padding
+                    implicitHeight: GridView.view.cellHeight - Style.current.padding
+                    title: model.name
+                    subTitle: StatusQUtils.Utils.elideText(model.address, 6, 4).replace("0x", "0×")
+                    asset.color: Utils.getColorForId(model.colorId)
+                    asset.emoji: model.emoji ?? ""
+                    asset.name: asset.emoji || "filled-account"
+                    asset.isLetterIdenticon: asset.emoji
+                    asset.letterSize: 14
+                    asset.bgColor: Theme.palette.primaryColor3
+                    asset.isImage: asset.emoji
+                    enabledNetworks: WalletStore.RootStore.filteredFlatModel // TODO: https://github.com/status-im/status-desktop/issues/14227
```

Comment:
We should try to avoid calling these singleton stores for components at this level. Better to pass a param.